cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -217,35 +213,28 @@ def verify_revert_snapshot(full_test_vm, provider, soft_assert, register_event,
     if provider.one_of(RHEVMProvider):
         full_test_vm.power_control_from_cfme(option=full_test_vm.POWER_OFF, cancel=False)
         full_test_vm.wait_for_vm_state_change(
-            desired_state=full_test_vm.STATE_OFF, timeout=900)
+            desired_state=full_test_vm.STATE_OFF, timeout=400)
 
     snapshot1.revert_to()
     # Wait for the snapshot to become active
     logger.info('Waiting for vm %s to become active', snapshot1.name)
-    wait_for(lambda: snapshot1.active, num_sec=300, delay=20, fail_func=provider.browser.refresh,
+    wait_for(lambda: snapshot1.active, num_sec=700, delay=30, fail_func=provider.browser.refresh,
              message="Waiting for the first snapshot to become active")
     # VM state after revert should be OFF
     full_test_vm.wait_for_vm_state_change(desired_state=full_test_vm.STATE_OFF, timeout=720)
     # Let's power it ON again
     full_test_vm.power_control_from_cfme(option=full_test_vm.POWER_ON, cancel=False)
-    full_test_vm.wait_for_vm_state_change(desired_state=full_test_vm.STATE_ON, timeout=900)
+    full_test_vm.wait_for_vm_state_change(desired_state=full_test_vm.STATE_ON, timeout=400)
     soft_assert(full_test_vm.mgmt.is_running, "vm not running")
     # Wait for successful ssh connection
-    wait_for(lambda: ssh_client.run_command('test -e snapshot1.txt').success,
-             num_sec=400, delay=10, handle_exception=True, fail_func=ssh_client.close(),
-             message="Waiting for successful SSH connection after revert")
-    try:
-        result = ssh_client.run_command('test -e snapshot1.txt')
-        assert result.success  # file found, RC=0
-        result = ssh_client.run_command('test -e snapshot2.txt')
-        assert result.failed  # file not found, RC=1
-        logger.info('Revert to snapshot %s successful', snapshot1.name)
-    except Exception:
-        logger.exception('Revert to snapshot %s Failed', snapshot1.name)
-    ssh_client.close()
+    with closing(connect_ssh(full_test_vm.mgmt, creds, num_sec=SSH_READY_TIMEOUT)) as ssh_client:
+        assert ssh_client.run_command('test -e snapshot1.txt').success
+        # This checks the exit status is 1 -- file doesn't exist.
+        assert ssh_client.run_command('test -e snapshot2.txt') == 1
 
 
 @pytest.mark.rhv1
+@pytest.mark.meta(blockers=[BZ(1805803, forced_streams=["5.10"])], automates=[1805803])
```

Comment:
@PJ Richardson I see we have both providers blocked by BZ blocker as is.

I think the problem is only with one of them and only with CFME 5.10.

I am not sure is there not a better way, but the most flexible way to "block" is having this somewhere (perhaps on top) of the test method:
```
if provider.one_of(RHEVProvider) and appliance.version.stream = "5.10":
  pytest.skip()
```

Note I didn't check the above code has no bugs, but I think you got the idea.

Reply:
First, this issue is in both 5.10 and 5.11. It's not consistent so you may get lucky and see some passing instances. Your suggestion would skip the rhv tests. There are a couple ways we can accomplish this functionality. As above, with an uncollect_if, or by changing the providers used for this test. My concern is that if we remove the blocker market, we won't have the metric data reflect tests blocked by the BZ. Unless I can tie in the BZ to  pytest.skip(). I'll get some input from the team.